showpass/Showpass-E2E-tests
# TA-83 Apply multiple categories simultaneously during event search.

**Description:**

This test scenario will be verifying that the following categories can be applied simultaneously during event search.

```
let categories = [
        [
          "Automotive, Boats, Air",
          "Button for Category: Automotive filter",
          "Button to remove Automotive, Boats, Air filter",
        ],
        [
          "Business & Professional",
          "Button for Category: Business filter",
          "Button to remove Business & Professional filter",
        ],
        [
          "Charity & Causes",
          "Button for Category: Charity filter",
          "Button to remove Charity & Causes filter",
        ],
      ];
```

Steps:

- Navigate to [Search Experiences](https://beta.showpass.com/s/events/all/) 
- Apply all 3 categories one after another (see above)
- Ensure the category has been applied

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-83

**How to test:**
Run the following spec and ensure all tests pass 'cypress/e2e/events/verify-filters.cy.js'

**Test execution results:**

<img width="755" alt="TA-83-all-tests-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/1f6bb0df-d8b3-48d5-bc00-54ef93d651a2">
